projects/src/main/java/org/odk/collect/projects/ProjectsRepository.kt

```diff
@@ -7,6 +7,8 @@ interface ProjectsRepository {
 
     fun add(project: Project)
 
+    fun update(project: Project)
```

Comment:
For the other repositories we've used just a `save` and then either added or updated based on whether the data object has an ID or not. Could we follow that pattern here, so it's consistent? I don't think one or the other is better, but it'd more work to reverse that decision for forms/instances than changing this now I think.

Reply:
Ok. Done.